contracts/bridge-v1.yml

```diff
@@ -646,6 +853,18 @@ components:
       properties:
         token:
           type: string
+    LinkConfirmationRequest:
+      type: object
+      properties:
+        requestId:
```

Comment:
this requestId == linkReferenceNumber from on-init?

Reply:
no. this is the requestId/Nonce. I forgot to add the "linkRefNumber" as part of the body of POST. updated now. Thanks for the catch.